tools/keytools/otp/Makefile

```diff
@@ -0,0 +1,41 @@
+-include ../../../.config
+-include ../../../tools/config.mk
+-include ../../../options.mk
+-include ../../../wcs/pkcs11.mk
+
+TARGET?=none
+ARCH?=ARM
+CROSS_COMPILE?=arm-none-eabi-
+CFLAGS+=-O0 -ggdb
+CFLAGS+=-I. -I../../../ -I../../../include
+CFLAGS+=-I./wcs
+CFLAGS+=-DFLASH_OTP_KEYSTORE -D__FLASH_OTP_PRIMER
+PRI_KS_OBJS+=startup.o otp-keystore-primer.o ../../../src/keystore.o
+LSCRIPT=target.ld
+LDFLAGS+=$(CFLAGS) -T$(LSCRIPT) -lc -Wl,-Map=otp-keystore-primer.map
+
+ifeq ($(TARGET),stm32h7)
+    CFLAGS+=-DTARGET_stm32h7
+    CFLAGS+=-mcpu=cortex-m7 -ffunction-sections -fdata-sections -fno-common -ffreestanding -nostartfiles
+    PRI_KS_OBJS+=../../../hal/stm32h7.o
+endif
+ifeq ($(TARGET),stm32h5)
+    CFLAGS+=-DTARGET_stm32h5
+    CFLAGS+=-mcpu=cortex-m33 -ffunction-sections -fdata-sections -fno-common -ffreestanding -nostartfiles
+    PRI_KS_OBJS+=../../../hal/stm32h5.o
+endif
+CC=$(CROSS_COMPILE)gcc
+OBJCOPY?=$(CROSS_COMPILE)objcopy
+SIZE?=$(CROSS_COMPILE)size
+
+
+otp-keystore-primer.bin: otp-keystore-primer.elf
+	@$(OBJCOPY) -O binary $(^) $(@)
+
+otp-keystore-primer.elf: $(PRI_KS_OBJS)
+	@$(CC) -o otp-keystore-primer.elf $(LDFLAGS) $(CFLAGS) $(PRI_KS_OBJS)
```

Comment:
Might consider the V=1 logic and `$(Q)` logic.

Got a warning about missing entry symbol `_start`. Still built fine, but is the default entry at 0x8000 really okay?

```
% make clean && make
arm-none-eabi-gcc -D"__WOLFBOOT" -Werror -Wextra -Wno-array-bounds -mthumb -mlittle-endian -mthumb-interwork -DARCH_ARM -Ihal -mcpu=cortex-m33 -DCORTEX_M33 -mcmse -DWOLFCRYPT_SECURE_MODE -DWOLFSSL_SP_ASM -DWOLFSSL_SP_ARM_CORTEX_M_ASM -DTZEN -DARCH_FLASH_OFFSET=0x08000000 -DWOLFBOOT_DUALBOOT -DWOLFBOOT_ORIGIN=0x0C000000 -DBOOTLOADER_PARTITION_SIZE= -DWOLFBOOT_ARCH_ARM -DTARGET_stm32h5 -D"FLASH_OTP_KEYSTORE" -D"WOLFBOOT_SIGN_ECC256" -D"FILL_BYTE=0xFF" -D"DUALBANK_SWAP=1" -D"NVM_FLASH_WRITEONCE" -Os -D"WOLFBOOT_NO_MPU" -DWOLFBOOT_VERSION=1 -DSECURE_PKCS11 -DWOLFSSL_PKCS11_RW_TOKENS -DCK_CALLABLE="__attribute__((cmse_nonsecure_entry))" -Ilib/wolfPKCS11 -DWP11_HASH_PIN_COST=3 -D"WOLFBOOT_HASH_SHA256" -DIMAGE_HEADER_SIZE=256  "-Wstack-usage=16688" -I"." -I"include/" -I"lib/wolfssl" -Wno-array-bounds -D"WOLFSSL_USER_SETTINGS" -D"WOLFTPM_USER_SETTINGS" -D"PLATFORM_stm32h5" -Wall -Wextra -Wno-main -ffreestanding -Wno-unused -nostartfiles -ffunction-sections -fdata-sections -fomit-frame-pointer   -mthumb -mlittle-endian -mthumb-interwork -mcpu=cortex-m33 -Wl,-gc-sections -Wl,-Map=wolfboot.map -ffreestanding -nostartfiles  tools/keytools/otp/otp-keystore-primer.c   -o tools/keytools/otp/otp-keystore-primer
/usr/local/Cellar/gcc-arm-none-eabi/20200630/bin/../lib/gcc/arm-none-eabi/9.3.1/../../../../arm-none-eabi/bin/ld: warning: cannot find entry symbol _start; defaulting to 0000000000008000
	[CC-ARM] hal/stm32h5.o
```

Reply:
This was an actual issue with otp-keystore-primer build, not selecting the right linker script